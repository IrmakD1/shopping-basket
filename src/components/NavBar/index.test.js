import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer';
import Navbar from '.';

describe('NavBar Component', () => {
    let component;
    let tree;
  
    beforeEach(() => {
      component = renderer.create(
        <div>
            <Router>
                <Navbar total={1}/>
            </Router>
        </div>
      );
      tree = component.toJSON();
    });
  
    test('it should render', () => {
      expect(tree).toMatchSnapshot();
    });
});
  