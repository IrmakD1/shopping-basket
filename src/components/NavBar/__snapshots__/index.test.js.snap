// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Component it should render 1`] = `
<div>
  <div
    className="nav-bar"
  >
    <ul>
      <li>
        <a
          href="/"
          onClick={[Function]}
        >
          <button
            className="button-is-secondary"
          >
            Food List
             
          </button>
        </a>
      </li>
      <li>
        <a
          href="/basket"
          onClick={[Function]}
        >
          <button
            className="button-is-secondary"
          >
            Basket:
             
            1
          </button>
        </a>
      </li>
    </ul>
  </div>
</div>
`;
